src/index.html

```diff
@@ -1,13 +1,44 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+
+  <body class="page">
+    <div class="card page__card" data-qa="card">
+      <img src="./images/imac.jpeg" alt="imac image" class="card__image">
+
+
+      <h3 class="card__description">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h3>
+
+      <p class="card__code">Код товара: 195434</p>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__reviews">Отзывов: 5</span>
+
+      </div>
+      <span class="card__price-container">
+        <div class="card__price">Цена:</div>
+        <span class="card__amount">69 999 грн</span>
+
+      </span>
+      <a class="card__buy" href="#buy" data-qa="hover">
```

Comment:
In this case also will be great if you write each attribute from a new line.

Reply:
I think it could be left here, but according to the case above, I corrected it to the same view, as you remarked